apps/messaging-app-extn-playground/shrinkwrap.yaml

```diff
@@ -0,0 +1,2452 @@
+dependencies:
```

Comment:
Is shrinkwrap.yaml still used by npm? This probably isn't needed 🤔

Reply:
Seems like it got added from Glitch. I will remove it